$NetBSD: patch-configure,v 1.21 2017/12/14 02:03:53 kamil Exp $

Use ncursesw6-config; define NCURSES_WIDECHAR to fix building on Darwin (at least).

--- configure.orig	2017-12-13 17:27:20.000000000 +0000
+++ configure
@@ -3165,12 +3165,13 @@ if test "$curses" != "no" ; then
     curses_inc_list="$($pkg_config --cflags ncurses 2>/dev/null):"
     curses_lib_list="$($pkg_config --libs ncurses 2>/dev/null):-lpdcurses"
   else
-    curses_inc_list="$($pkg_config --cflags ncursesw 2>/dev/null):-I/usr/include/ncursesw:"
-    curses_lib_list="$($pkg_config --libs ncursesw 2>/dev/null):-lncursesw:-lcursesw"
+    curses_inc_list="$(@PREFIX@/bin/ncursesw6-config --cflags 2>/dev/null):-I/usr/include/ncursesw:"
+    curses_lib_list="$(@PREFIX@/bin/ncursesw6-config --libs 2>/dev/null):-lncursesw:-lcursesw"
   fi
   curses_found=no
   cat > $TMPC << EOF
 #include <locale.h>
+#define NCURSES_WIDECHAR 1
 #include <curses.h>
 #include <wchar.h>
 int main(void) {
